prow/plugins/assign/assign.go

```diff
@@ -98,6 +106,21 @@ func parseLogins(text string) []string {
 	return parts
 }
 
+// Check if members are member of org.
+// Return a list of non-org members
+func CheckMembership(gc githubClient, log *logrus.Entry, org string, members []string) []string {
```

Comment:
Also, note that this uses one API token per member. This isn't a bad thing since we expect that there will only be a couple members per call.